### ISSUE TITLE
[feature request] Keep errors open in a separate window

### ISSUE THREAD

**Issue by mnbf9rca** [DamnWidget/anaconda#592]
When reviewing code with a lot of errors (someone else's of course :) ) it's annoying when the sublime popup with the error list keeps disappearing. It would be good to have it as a floating window, or docked to the bottom of the editor, like the error list in Visual Studio when working with e.g. c#. Is this possible?

**Comment by DamnWidget** [comment]
Sadly, ST3 API does not offers too many things to play with, the only floating windows are the popups and they disappear as soon as a key is pressed. Then we have phantoms, but those are attached to a specific `Region` (line and column) so they don't follow you if you scroll the window.

It is not possible to dock the panel (that would be actually cool), so the only thing that comes to my mind is use some sort of bottom panel but that is not trivial or fast.

**Comment by mnbf9rca** [comment]
hmm - that's a real shame. i wonder what else might help. Perhaps if they were available as tooltips? And a keyboard shortcut to the full list (I couldnt find one)?

**Comment by DamnWidget** [comment]
Nope, there are no shorcuts because I consider them invasive, I should really remove all the defined anaconda shortcuts so the user can add whatever they like.

The command to run from a shortcut is `anaconda_get_lines`

**Comment by mnbf9rca** [comment]
personally, i find shortcuts really help - especially if it means i can avoid taking my hands off the keyboard to use the mouse. I've added a custom one now, so thanks for the pointer. I'd appreciate if you can consider how this feature request might be implemented e.g. if in the future Jon Skinner decides to add something to sublime to enable it.

**Comment by DamnWidget** [comment]
Yes, they help, I don't use the mouse neither, I use the `Command Palette` to invoke actions but there are a lot of possibilities that I override any keybinding that other plugin, window manager or Operating System is already using so I prefer the user takes control over key bindings.

I could think about something to possibly support this request but currently I am pretty busy.

**Comment by mnbf9rca** [comment]
thanks
